### PR TITLE
Implement to do item changes

### PR DESCRIPTION
**Pull request by mpiimamahmoud**
**What does this pr do?**
- it implements a model class

### CODE REVIEW

File: Todo.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (huxaiphaer) [inline Todo.csproj:1]: Why did you update this code ?

Comment (mpiimamahmoud) [reply Todo.csproj:1]: It was done automatically****

**Review by huxaiphaer — APPROVED** [review]
LGTM (Looks Good To Me)

File: Todo.csproj (hunk 1 above)
Comment (sharifmuyanja) [reply Todo.csproj:1]: good code so far

**Review by hammshx — COMMENTED** [review]
Some comments needed to show purpose of class

**Review by malfahad — APPROVED** [review]
LGTM

**Review by sharifmuyanja — APPROVED** [review]
ka sukali keko

File: Models/TodoItem.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Todo.Models
+{
+    public class TodoItem
+    {
+        public long id { get; set; }
+        public string Name { get; set; }
+        public bool IsComplete { get; set; }
```
Comment (malfahad) [inline Models/TodoItem.cs:12]: Add a description field

**Review by malfahad — CHANGES_REQUESTED** [review]

**Review by malfahad — APPROVED** [review]

**Pull request merged**